packages/tentales/src/middleware/__tests__/middleware-utils.test.ts

```diff
@@ -0,0 +1,7 @@
+import { createTestConfig } from "../../__test-helpers__"
+import { createConfigHooks } from "../middleware-utils"
+
+test("It create config hooks", async () => {
+  const resp = createConfigHooks(createTestConfig())
+  expect(resp).toMatchSnapshot()
```

Comment:
The snapshot here is very sparse with only `Array []`..

Reply:
Good catch!